docs/_articles/en/v3.x/release-information.md

```diff
@@ -12,6 +12,32 @@ We publish the latest Code Analyzer monthly.
 * Run `sfdx plugins:update` and `sfdx plugins:update --help` to update Code Analyzer and help to the latest version.
 * Follow these [instructions](./en/v3.x/getting-started/install/#upgrade-plug-in) to update Code Analyzer
 
+## [v3.6.0](https://github.com/forcedotcom/sfdx-scanner/tree/v3.6.0) (10-25-2022)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v3.5.1...v3.6.0)
+
+### Release Summary
+
+* NEW: SFCA v3.x will now publish as `latest`.
+* CHANGE \[SFGE\]: `scanner:run:dfa` has a new `--rule-disable-warning-violation` flag to disable warnings on StripInaccessible call with AccessType.READABLE.
+* CHANGE \[SFGE\]: `--ignore-parse-errors` option has been removed from `scanner:run:dfa`.
+* CHANGE: Help text and user interface messages have been updated.
+* NEW: PMD version has been updated to 6.50.0.
```

Comment:
"CHANGE" as per PR at line 35.

Reply:
We've been calling out PMD upgrades as NEW in previous releases. Will change line 35 to match.